web-ng/src/app/services/observation/observation.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Feature } from './../../shared/models/feature.model';
+import { Project } from './../../shared/models/project.model';
```

Comment:
Move imports below comments (the VSCode plugin sometimes misplaces the auto-imports :/)